File: xml/jruby/read/xml_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	read/xml_read.rb
#
#					May/18/2011
#
# -------------------------------------------------------------
include	Java
import	java.lang.System
#
require "rexml/document"
include REXML
load '/var/www/data_base/common/ruby_common/file_io.rb'
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
load '/var/www/data_base/common/ruby_common/xml_manipulate.rb'
# -------------------------------------------------------------
#
System.out.println("*** 開始 ***")
#
xml_file = ARGV[0]
#
xml_str = read_string_proc(xml_file)

dict_aa=xml_to_dict_proc(xml_str)
dict_display_proc(dict_aa)
#
System.out.println("*** 終了 ***")
#
# -------------------------------------------------------------

File: xml/ringo/read/xml_read.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	xml_read.js
//
//						Dec/27/2013
//
// ------------------------------------------------------------------
var fs= require ("fs");
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
var xml_manipulate = require ('/var/www/data_base/common/ringo_common/xml_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");

var args = require('system').args;
var file_in = args[1];

var str_in = fs.read(file_in);

var dict_aa =  xml_manipulate.xml_to_dict_proc (str_in);

text_manipulate.dict_display_proc (dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------
